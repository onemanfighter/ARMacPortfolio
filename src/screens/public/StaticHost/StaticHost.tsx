import { StaticHostProps } from './type';

const StaticHost = (props: StaticHostProps) => {
  return (
    <div>
      <h1>StaticHost</h1>
    </div>
  );
};

export default StaticHost;
