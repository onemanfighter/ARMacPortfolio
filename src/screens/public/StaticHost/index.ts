export { default as StaticHost } from './StaticHost';
