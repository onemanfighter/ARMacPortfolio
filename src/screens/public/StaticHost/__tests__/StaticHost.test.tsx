import { render } from '@testing-library/react';
import StaticHost from '../StaticHost';

describe('StaticHost', () => {
  it('should render correctly to match snapshot', () => {
    const { container } = render(<StaticHost />);

    expect(container).toMatchSnapshot();
  });
});
