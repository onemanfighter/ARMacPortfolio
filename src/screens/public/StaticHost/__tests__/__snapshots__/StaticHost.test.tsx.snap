// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StaticHost should render correctly to match snapshot 1`] = `
<div>
  <div>
    <h1>
      StaticHost
    </h1>
  </div>
</div>
`;
